src/metabase/lib/schema/expression/string.cljc

```diff
@@ -37,8 +37,8 @@
 
 (mbql-clause/define-tuple-mbql-clause :split-part :- :type/Text
   [:schema [:ref ::expression/string]]
-  [:schema [:ref ::expression/string]]
-  [:schema [:ref ::expression/integer]])
+  [:schema [:string {:min 1}]] ;; literal string
```

Comment:
nit: personally I find the comment labels like in `:replace` above helpful, i.e. `#_delimiter` etc.